src/commcare_cloud/ansible/roles/deploy_hq/tasks/staticfiles_compress.yml

```diff
@@ -0,0 +1,37 @@
+- name: Compress static files
+  command:
+    cmd: ./manage.py {{ item }}
+    chdir: '{{ code_source }}'
+  register: stop_loop_hack
+  when: not stop_loop_hack.failed | default(False)
+  loop:
+    - compress --force -v 0
+    - purge_compressed_files
+
+- name: Push shared staticfiles cache
+  when: shared_dir_for_staticfiles
+  block:
+    - name: Create shared directory
+      become: true
+      become_user: root
+      file:
+        path: '{{ shared_dir_for_staticfiles }}/{{ code_version }}/staticfiles/CACHE'
+        state: directory
+        owner: '{{ cchq_user }}'
+        group: '{{ cchq_user }}'
+        mode: 0755
+
+    - name: Copy staticfiles/CACHE to shared dir
+      synchronize:
+        src: '{{ code_source }}/staticfiles/CACHE/'
+        dest: '{{ shared_dir_for_staticfiles }}/{{ code_version }}/staticfiles/CACHE/'
+        recursive: true
+        delete: true
+      delegate_to: '{{ inventory_hostname }}'
```

Comment:
This `rsync` command is taking a slightly different form than it had previously. For reference:

https://github.com/dimagi/commcare-cloud/blob/b1f5d2c41583f93d8a3d43ba1707825f8ced78e3/src/commcare_cloud/fab/operations/staticfiles.py#L68-L74

I think it will have the same intended outcome, although the previous command would have synchronized `staticfiles` and `CACHE` directories throughout the repo. To my knowledge none should exist other than `staticfiles/CACHE` in the root of the repo. Please review carefully and raise an issue if you think it changed in some other meaningful way.

Reply:
Old/unused files at `{{ shared_dir_for_staticfiles }}/{{ code_version }}` are never removed. I'm adding a note to add that to "clean releases" logic when I get around to bringing that over to Ansible.

Reply:
I don't see any issue with removing old staticfiles (provided there is some overlap around the deploy).